File: C++/HerenciaClase/Profesor.cc

#include "Profesor.h"
#include <iostream>
using namespace std;
Profesor::Profesor( int edad, int cursos):Persona(edad){
	cursos = cursos;
}

void Profesor::presentacion(){
	cout << "Hola  tengo " << edad << " años.\n";
	cout << "Enseño " << cursos << " cursos.\n";
}

File: Java/HerenciaClase/Profesor.java
package HerenciaClase;

public class Profesor extends Persona{
	private int cursos; //cantidad de cursos que enseña
	
	public Profesor (String nombre, String apellido, int edad, int cursos) {
		super(nombre,apellido,edad);
		this.cursos = cursos;
	}
	@Override
	public String presentación() {
		return "Hola soy un profesor, mi nombre es " + getNombre() + " "+ getApellido()
		+ " y tengo " + getEdad() + " años, enseño "+ cursos + " cursos ";
	}
	

}
